src/config.rs

```diff
@@ -13,6 +13,7 @@ use crate::{
 };
 
 pub const DEFAULT_SIDEBAR_WIDTH: f64 = 140.0;
+pub const DEFAULT_WINDOW_SIZE: (f64, f64) = (800.0, 350.0);
```

Comment:
At this point it might be nicer to just implement `Default` for `WindowSettings`?

Reply:
Good idea.